### ISSUE TITLE
Minification can result in malformed selectors

### ISSUE THREAD

**Issue by lozcalver** [bigfork/gulp-handler#1]
Example output:

``` css
.modern .genform .text label,.modern
    .genform .textarea label,.modern #OmarBoutique_MemberLoginForm_LoginForm .text label,.modern
    #ChangePasswordForm_ChangePasswordForm .text label,.modern
    #OmarBoutique_MemberLoginForm_LostPasswordForm .text label,.modern #OmarBoutique_MemberLoginForm_LoginForm .textarea label,.modern
    #ChangePasswordForm_ChangePasswordForm .textarea label,.modern
    #OmarBoutique_MemberLoginForm_LostPasswordForm .textarea label {
  display: none;
}
```

I don’t know why sass puts the line breaks in such weird places, but it looks like one of the regexes is removing the new line but not adding a space

**Comment by stnvh** [comment]
Hmm, I've ran into blips with our minifier before too. We should use a maintained plugin like [cssnano](http://cssnano.co). less maintenance and head-scratching.

**Comment by stnvh** [comment]
Indirectly addressed a while back in shoelace - [61bf176](https://github.com/bigfork/SilverStripe-Shoelace/commit/61bf1769291fdff36e4d0610c304423667ca6ac4)
